examples/finding_best_open_source_embedding_model/compose.yaml

```diff
@@ -0,0 +1,20 @@
+name: pgai-ollama
+services:
+  db:
+    image: timescale/timescaledb-ha:pg17
+    environment:
+      POSTGRES_PASSWORD: postgres
+      OPENAI_API_KEY: "sk-proj..."
+    ports:
+      - "5432:5432"
+    volumes:
+      - ./data:/var/lib/postgresql/data
+  vectorizer-worker:
+    image: timescale/pgai-vectorizer-worker:v0.2.1
+    environment:
+      PGAI_VECTORIZER_WORKER_DB_URL: postgres://postgres:postgres@db:5432/postgres
+      OPENAI_API_KEY: "sk-proj..."
```

Comment:
You only need to set the API_KEY on the DB if you don't create an openai vectorizer, which I think the notebook doesn't do so maybe omit this line.